### DISCUSSION TITLE
How to read connection info from datasource center in java or spark?

### DISCUSSION THREAD

**Discussion by ziya05** [apache/dolphinscheduler#16337] [Q&A] [answered]
For example, Java needs to read the contents of database 1 and calculate, and then store the calculation results in database 2. Currently database 1 and database 2 have been configured in the data source center of dolphinscheduler. How can it be read in the java program?

**Comment by SbloodyS** [comment, marked as answer]
This feature is not currently supported.

**Reply by ziya05** [reply]
Thanks for your reply
